IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormPullTests.cs

```diff
@@ -207,6 +207,35 @@ public void Should_remember_user_choice_upon_pull(bool autoStashChecked)
                 AppSettings.PullAction.Merge);
         }
 
+        [Test]
+        public void Should_disable_enable_merge_options_when_no_tracking()
+        {
+            RunFormTest(
+                form =>
+                {
+                    /* Since we're using _commands.StartPullDialog, later FormPullLoad gets called and
+                       in it we call DisableMergeOptionsWhenCurrentBranchIsNotTracking(),
+                       so no need to call it again after the accessor initialization.
+                       Thus, control states are:
+                       Rebase.Enabled = false
+                       Merge.Enabled = false
+                       Fetch.Enabled = true
+                       Fetch.Checked = true
+                    */
+                    var accessor = form.GetTestAccessor();
+
+                    accessor.Merge.Enabled.Should().Be(false);
+                    accessor.Merge.Checked.Should().Be(false);
+                    accessor.Rebase.Enabled.Should().Be(false);
+                    accessor.Rebase.Checked.Should().Be(false);
+                    accessor.Fetch.Enabled.Should().Be(true);
+                    accessor.Fetch.Checked.Should().Be(true);
+                },
```

Comment:
Looking at this test it is totally not obvious how we ended up here.

Do we need to also assert the remote and the local branches are empty?
What would happen if a user types in the remote and/or the local anything?
![image](https://user-images.githubusercontent.com/20372363/96333924-885dd180-1075-11eb-8afd-1879e48abf5d.png)

"Disable merge rebase commands when branch is local" implies a bigger context than checking few checkboxes - it involves the the remote and the local branches, and the test must assert those before and after. And we also need test when either/or are specified.

Reply:
To me it looked that it's just that.

Reply:
I'd rather extend the test whose setup needs to be extended above.

Reply:
> To me it looked that it's just that.

Could you please explain?

To me "_Disable merge rebase commands when branch is local_" there are few players here:
- **rebase command** - these you have disabled
- **local branch**
- **remote branch**

Right now I don't believe this is working correctly. E.g. I created a branch on a random commit (that doesn't have a remote tracking branch associated), and opened the Pull dialog:
![ge-local-branch](https://user-images.githubusercontent.com/4403806/96393947-71051c80-120c-11eb-99ac-11c09f9d0cb8.gif)